apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/GrpcRdaSource.java

```diff
@@ -113,7 +114,7 @@ public GrpcRdaSource(
    * @throws ProcessingException wrapper around any Exception thrown by the service or sink
    */
   @Override
-  public int retrieveAndProcessObjects(int maxPerBatch, RdaSink<TResponse> sink)
+  public int retrieveAndProcessObjects(int maxPerBatch, RdaSink<TMessage, TClaim> sink)
```

Comment:
This seems like a very complex method, curious if this can be broken up a little for better readability

Reply:
It's all very linear.  It hasn't changed much from master and since it's mostly catch blocks to carry through the correct processed count in a ProcessingException it can't be separated out much.  I tried and the separate method just contains the about the same set of catch blocks as its caller.  I suggest we consider that as a refactoring of its own for a future ticket.